docs/separating-entity-definition.md

```diff
@@ -131,3 +137,73 @@ export const CategoryEntity = new EntitySchema<Category>({
 });
 ```
 
+## Extending Schemas
+
+When using the `Decorator` approach it is easy to `extend` basic columns to an abstract class and simply extend this. 
+For example, your `id`, `createdAt` and `updatedAt` columns may be defined in such a `BaseEntity`. For more details, see 
+the documentation on [concrete table inheritance](entity-inheritance.md#concrete-table-inheritance).
+
+When using the `EntitySchema` approach, this is not possible. However, you can use the `Spread Operator` (`...`) to your 
+advantage.
+
+Reconsider the `Category` example from above. You may want to `extract` basic column descriptions and reuse it across 
+your other schemas. This may be done in the following way:
+
+```ts
+import {EntitySchemaColumnOptions} from "typeorm";
+
+export const BaseColumnSchemaPart = {
+  id: {
+    type: Number,
+    primary: true,
+    generated: true,
+  } as EntitySchemaColumnOptions,
+  createdAt: {
+    name: 'created_at',
+    type: 'timestamp with time zone',
+    createDate: true,
+  } as EntitySchemaColumnOptions,
+  updatedAt: {
+    name: 'updated_at',
+    type: 'timestamp with time zone',
+    updateDate: true,
+  } as EntitySchemaColumnOptions,
+};
+```
+
+Now you can use the `BaseColumnSchemaPart` in your other schema models, like this:
+
+```ts
+export const CategoryEntity = new EntitySchema<Category>({
+    name: "category",
+    columns: {
+        ...BaseColumnSchemaPart,    
+        // the CategoryEntity now has the defined id, createdAt, updatedAt columns!
+        // in addition, the following NEW fields are defined
+        name: {
+            type: String
+        }
+    }
+});
+```
+
+Be sure to add the `extended` columns also to the `Categeory` interface (e.g., via `export interface Category extend BaseEntity`).
+
+## Using Schemas to Query / Insert Data
+
+Of course, you can use the defined schemas in your repositories or entity manager as you would use the decorators.
+Consider the previously defined `Category` example (with its `Interface` and `CategoryEntity` schema in order to get 
+some data or manipulate the database.
+
+```ts
+// request data
+const categoryRepository = getRepository<Category>('CategoryEntity');
```

Comment:
actually what you need to supply isn't `'CategoryEntity'`, but `CategoryEntity`, e.g. 

```ts
const categoryRepository = getRepository(CategoryEntity); // type is inferred
```

string support is legacy and it might be removed in the future.

Reply:
I will check this tomorrow and give feedback on this.. Thanks a lot for your helpful comments!

Reply:
i pushed a new commit with the changes